packages/ckeditor5-image/src/image/imageinlinetoblockcommand.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module image/image/imageinlinetoblockcommand
+ */
+
+import Command from '@ckeditor/ckeditor5-core/src/command';
+import { insertImage, isImageInline } from './utils';
+
+/**
+ * The image inline command. It is used to convert block image io inline image.
+ *
+ * @extends module:core/command~Command
+ */
+export default class ImageInlineToBlockCommand extends Command {
+	/**
+	 * @inheritDoc
+	 */
+	refresh() {
+		const element = this.editor.model.document.selection.getSelectedElement();
+
+		this.isEnabled = isImageInline( element );
```

Comment:
Maybe this command should be disabled if a block image is not allowed by the schema?

Reply:
answered here: https://github.com/ckeditor/ckeditor5/pull/8726#discussion_r550138725

Reply:
updated comment linked above (similar situation)